.github/workflows/linux.yml

```diff
@@ -59,6 +59,7 @@ env:
   QUICTLS_VERSION: 3.3.0
   # renovate: datasource=github-tags depName=rustls/rustls-ffi versioning=semver registryUrl=https://github.com
   RUSTLS_VERSION: 0.15.0
+  OPENLDAP_VERSION: 2.6.10
```

Comment:
Could this use a renovate rule for automatic bumps?

Reply:
have now added a Renovate rule that should match this.